### PR TITLE
Use GitHub Actions for ROS2 CI

### PR DESCRIPTION
**Pull request by mm318**
This pull request removes the use of Travis and replaces it with the use of GitHub Actions.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by raghaprasad — COMMENTED** [review]
We are going to retain 2 sample applications it seems. 
Better to move all the GitHub actions logic into its own repo?

**Comment by mm318** [comment]
> Better to move all the GitHub actions logic into its own repo?

Yes, that is a good idea. There are three actions under this sample app's `.github/actions` directory, so we should make 3 additional repos under `aws-robotics`.

**Comment by mm318** [comment]
Oops, I missed a commit in this pull request earlier.

File: robot_ws/src/cloudwatch_robot/package.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version='1.0' encoding='utf-8'?>
 <package format="3">
   <name>cloudwatch_robot</name>
-  <version>2.0.0</version>
+  <version>2.0.1</version>
```
Comment (mm318) [inline robot_ws/src/cloudwatch_robot/package.xml:4]: So I take it based on https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/pull/69#discussion_r410378567, this should also be `2.1.0`?

Comment (raghaprasad) [reply robot_ws/src/cloudwatch_robot/package.xml:4]: Yup!

**Comment by raghaprasad** [comment]
> > Better to move all the GitHub actions logic into its own repo?
> 
> Yes, that is a good idea. There are three actions under this sample app's `.github/actions` directory, so we should make 3 additional repos under `aws-robotics`.

Sounds good to me. 
Can we cut a ticket with OS team? also we might be potentially be able to reuse those repo's for CE ci's ?

**Pull request merged**